js/ui/grid_core/filtering/utils.js

```diff
@@ -0,0 +1,152 @@
+// @ts-check
+
+import { Deferred } from '../../../core/utils/deferred';
+import DataSource from '../../../data/data_source';
+import { normalizeDataSourceOptions } from '../../../data/data_source/utils';
+import variableWrapper from '../../../core/utils/variable_wrapper';
+import { isFunction, isString } from '../../../core/utils/type';
+import gridCoreUtils from '../ui.grid_core.utils';
+
+function normalizeGroupingLoadOptions(group) {
+    if(!Array.isArray(group)) {
+        group = [group];
+    }
+
+    return group.map((item, i) => {
+        if(isString(item)) {
+            return {
+                selector: item,
+                isExpanded: i < group.length - 1,
+            };
+        }
+
+        return item;
+    });
```

Comment:
Minor comment.

Let's write it shortly:
```suggestion
        return group.map((item, idx) =>
        isString(item)
            ? ({ selector: item, isExpanded: idx < group.length - 1 })
            : item);
```